meta-lmp-support/recipes-core/initrdscripts/initramfs-framework/ostreecheck

```diff
@@ -15,9 +15,14 @@ ostreecheck_enabled() {
 }
 
 ostreecheck_run() {
-	msg "OSTree check here"
-	ls -l /
-	ls -l /sysroot
+	# We are running between ostree-prepare-root /rootfs and the move of
+	# that to /.
+	# In ostree 2021.1, ostree-prepare-root creates /run/ostree-booted, which
+	# confuses ostree commands run before the remount - they try to remount
+	# /sysroot, which doesn't exist yet. Remove this to make it work.
+	rm /run/ostree-booted
```

Comment:
Should there be some kind of "backwards compatibilty / future proofing"? I.e. check if the fiel exists and only then remove it? Would it fail it the file didn't exist?

Reply:
This would print an error if it didn't exist, but continue.

Probably safer to move it out of the way then put it back (and send errors to `/dev/null`). I'll do that. Our initramfs `/run` actually becomes the real `/run` later, it's not chucked. But then something later in boot currently recreates `/rev/ostree-booted` manually anyway, so my deletion still works for now.